microsoft-edge/toc.yml

```
@@ -424,6 +424,10 @@
                   href: devtools-guide-chromium/rendering-tools/js-runtime.md
                   displayName: Memory tool
 
+                - name: Share memory traces
+                  href: devtools-guide-chromium/memory-problems/share-traces.md
+                  displayName: Memory tool
```

```suggestion
                  displayName: Memory tool, Memory panel
```
something to consider: will ppl try to look up the term 'panel' for all the articles?  consider a PR adding 'panel' index entry synonym on all "tool" pages.

---

Sounds like a good experiment for us to try in a separate PR.